src/Widgets/TaskRow.vala

```diff
@@ -75,7 +75,7 @@ public class Tasks.TaskRow : Gtk.ListBoxRow {
 
         check = new Gtk.CheckButton ();
         check.valign = Gtk.Align.CENTER;
-        Tasks.Application.set_task_color (source, check);
+        check.get_style_context ().add_provider (taskrow_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
```

Comment:
Is this necessary now?

Reply:
@danrabbit oh boy, seems like it was already late... Good catch! Of course this is no longer needed. Removed.